templates/torrc.j2

```diff
@@ -87,7 +87,7 @@ DirPortFrontPage {{ torrc_dir_port_front_page }}
 MyFamily {{ torrc_my_family }}
 {% endif %}
 
-{%- if torrc_exit_relay %}
+{%- if torrc_or_ports %}
```

Comment:
All the other parameters are rendered if they are defined. We are setting a default value for the variables. What do you think about unconditionally render the parameter?

Reply:
Not all of them! See for example `PublishServerDescriptor` or `CookieAuthentication`. Namely: every option is rendered only when "it is meaningful" (except for `RunAsDaemon` which is rendered due to my lazyness to understand all the jinja2 whitespace quirks).

I'd rather keep the final torrc files as short as possible to the extreme of an empty file for a private socks5-only tor service.

Reply:
:+1: